pkg/node/node.go

```diff
@@ -150,6 +151,19 @@ func NewBee(o Options) (*Bee, error) {
 	}
 	b.p2pService = p2ps
 
+	// wait for nat manager to init
+	logger.Debug("initializing NAT manager")
+	select {
+	case <-p2ps.NATManager().Ready():
+		logger.Debug("NAT manager initialized")
+	case <-time.After(10 * time.Second):
+		logger.Warning("NAT manager init timeout")
+	}
+
+	// this is magic sleep to give NAT time to sync the mappings
+	// this is a hack, kind of alchemy and should be improved
+	time.Sleep(3 * time.Second)
```

Comment:
I would put this sleep right after the Ready() channel reads and before the `NAT manager initialized` log line. If the timeout happens, there is no need to wait more and there is a delay after the `NAT manager initialized` log message.

Reply:
Correct! 🙈